package corejava;

public class exceptionhandling {

	public static void main(String[] args)  {
		System.out.println("jjiejt");
		try {
		Thread.sleep(4000);
		}
		catch(Exception e) {
			System.out.println("hello");
		}
		System.out.println("hujkrf44");
		        int a = 5;
		        int b = 0;
		        try {
		        System.out.println(a/b);
		        }
		    /*    catch (Exception e) {
		        	System.out.println("No problem");
		        }*/
		        finally {
		        	System.out.println("hello");
		        	
		        }
	}

}
